gwnr/waveform/hybridize.py

```diff
@@ -0,0 +1,177 @@
+def perform_hybridisation(inspiral, merger_ringdown, frq_attach, frq_width):
+    
+    
+    timeaxis_insp = np.arange(len(inspiral))*dt
+    timeaxis_mr = np.arange(len(merger_ringdown))*dt
+    
+    
+    ''' Computing waveform attributes: amplitude, phase, frequency (no interpolation) '''
+
+    def compute_amplitude(waveform):
```

Comment:
Since no other operation is using PyCBC functions, I believe it is better for us to stick to numpy for these computations to 
1. Keep the imports to a minimum  
2. Keep the algorithm implementation less specific to waveforms  
3. PyCBC also uses the same technique to compute amplitude/phase etc, but if we use numpy we have the liberty to specify the kind of derivatives we want to use